src/prototype_room_attack.js

```diff
@@ -108,14 +126,17 @@ Room.prototype.attackRoom = function() {
 
   addRoom(player, this);
 
-  if (player.level === 0) {
+  if (this.controller.level < 3 && player.level === 0) {
```

Comment:
Hm, if this is the reason we should check for towers :-)

Still a `yes`, no so much resources and the opponent could underestimate us.